deploy/cdk/bin/base.ts

```diff
@@ -39,23 +41,46 @@ const contact = getRequiredContext('contact')
 const namespace = getRequiredContext('namespace')
 const envName = getRequiredContext('env')
 const contextEnv = getRequiredContext('environments')[envName]
-if(contextEnv === undefined || contextEnv === null)
+if(contextEnv === undefined || contextEnv === null) {
   throw new Error(`Context key 'environments.${envName}' is required.`)
+}
 
 // The environment objects defined in our context are a mixture of properties.
 // Need to decompose these into a cdk env object and other required stack props
 const env = { account: contextEnv.account, region: contextEnv.region, name: envName }
 const { useVpcId, domainName, createDns, useExistingDnsZone, slackNotifyStackName } = contextEnv
 
 const oauthTokenPath = app.node.tryGetContext('oauthTokenPath')
+const projectName = getRequiredContext('projectName')
+const description = getRequiredContext('description')
 
-const foundationStack = new FoundationStack(app, `${namespace}-foundation`, {
+const foundationStack = new FoundationStack(app, `${projectName}-foundation`, {
```

Comment:
I think we should still keep with a single namespace instead of breaking it up into projectName-namespace. The single namespace idea was a way of reducing required overrides when deploying since things like project-stage were in most cases only ever used to concatenate and create namespaced stacks.

For handling the case where we have two front ends deployed by this same app, we just need to instantiate both of those in base with different stack names, but both would still just use namespace as the prefix